components/01-globals/components.scss

```diff
@@ -0,0 +1,5 @@
+// Mixins
+
+@import 'mixins/_grid-column-width.scss';
```

Comment:
We are following `clean-imports` sass lint rules - https://github.com/sasstools/sass-lint/blob/master/docs/rules/clean-import-paths.md

Reply:
BTW. We need to add these linters here.